kura/test/org.eclipse.kura.core.keystore.test/src/test/java/org/eclipse/kura/core/keystore/FilesystemKeystoreServiceImplTest.java

```diff
@@ -99,6 +101,10 @@ public void setupDefaultKeystore()
         try (OutputStream os = new FileOutputStream(STORE_PATH)) {
             this.store.store(os, STORE_PASS.toCharArray());
         }
+
+        // clean test keystores
+        Files.deleteIfExists(Paths.get(NEW_STORE_PATH));
+        Files.deleteIfExists(Paths.get(UPDATED_NEW_STORE_PATH));
```

Comment:
Actually I would use a TearDown method so that the filesystem is cleaned up after each unit test leveraging the `@After` annotation provided by junit.

Something like:

```java
@After
public void tearDown() throws Exception {
    Files.deleteIfExists(Path.get(...));
}
```

Does this make sense to you?

Reply:
i'd leave as it is